test/load/locustfile-performance.py

```diff
@@ -0,0 +1,73 @@
+import os
+import time
+import binascii
+
+from locust import Locust, TaskSet, events, task
+
+import grpc.experimental.gevent as grpc_gevent
+grpc_gevent.init_gevent()
+
+import grpc
+from irohalib import Iroha, IrohaGrpc
```

Comment:
Let's move to Iroha from pypi.
```
pip install iroha
```
and
```
from iroha import Iroha, IrohaGrpc, IrohaCrypto
```

The second preparation step from test/load/README.md will be much simpler.